### PR TITLE
Feature/1841/remove button from mui

### PR DESCRIPTION
**Pull request by mhevyk**
- Removed all styles for button from mui theme
- Added few styles to `Button` from design system
- Added `Danger` and `Base` button variants
- Replaced `Button` components with `AppButton`

### CODE REVIEW

File: src/components/enhanced-table/enhanced-table-pagination/EnhancedTablePagination.jsx [hunk 1]
```diff
@@ -70,13 +71,13 @@ const EnhancedTablePagination = ({ pagination }) => {
           type='number'
           value={pageInput}
         />
-        <Button
+        <AppButton
           onClick={() => handlePageSubmit(pageCount)}
           sx={styles.btn}
-          variant='outlined'
+          variant={ButtonVariantEnum.Contained}
```
Comment (asya982) [inline src/components/enhanced-table/enhanced-table-pagination/EnhancedTablePagination.jsx:77]: Isn't `contained` not the same as `outlined`?

File: src/components/file-uploader/FileUploader.tsx [hunk 2]
```diff
@@ -21,6 +20,7 @@ import {
   InputEnum,
   SizeEnum
 } from '~/types'
+import AppButton from '../app-button/AppButton'
```
Comment (asya982) [inline src/components/file-uploader/FileUploader.tsx:23]: Use the right import😅

File: src/components/search-autocomplete/SearchAutocomplete.tsx [hunk 3]
```diff
@@ -30,6 +29,7 @@ import {
   VisibilityEnum,
   TextFieldVariantEnum
 } from '~/types'
+import AppButton from '../app-button/AppButton'
```
Comment (asya982) [inline src/components/search-autocomplete/SearchAutocomplete.tsx:32]: Same here

File: src/containers/edit-profile/profile-tab/profile-tabs/profile-general-tab/ProfileGeneralTab.tsx [hunk 4]
```diff
@@ -142,12 +142,13 @@ const ProfileGeneralTab: FC<ProfileTabProps> = ({ user }) => {
...
               buttonText={t('editProfilePage.profile.generalTab.uploadTitle')}
               emitter={addPhoto}
+              size={SizeEnum.Large}
               validationData={validationData}
               variant={ButtonVariantEnum.ContainedLight}
             />
             <AppButton
               onClick={handleRemovePhoto}
-              size={SizeEnum.Medium}
+              size={SizeEnum.Large}
```
Comment (asya982) [inline src/containers/edit-profile/profile-tab/profile-tabs/profile-general-tab/ProfileGeneralTab.tsx:151]: Why here sizes are different?

File: src/components/enhanced-table/enhanced-table-pagination/EnhancedTablePagination.jsx (hunk 1 above)
Comment (mhevyk) [reply src/components/enhanced-table/enhanced-table-pagination/EnhancedTablePagination.jsx:77]: `Outlined` button variant was deleted from theme completely (discussed with designer). I decided to use `contained` as default for now and also this component is used on admin dashboard and it is not our priority right now.

File: src/components/file-uploader/FileUploader.tsx (hunk 2 above)
Comment (mhevyk) [reply src/components/file-uploader/FileUploader.tsx]: Thanks a lot, fixed

**Review by asya982 — APPROVED** [review]

**Review by OliaHrytsak — APPROVED** [review]

**Review by abalanovsky — APPROVED** [review]

File: src/containers/guest-home-page/reset-password/ResetPassword.jsx [hunk 5]
```diff
@@ -21,6 +20,8 @@ import { confirmPassword, password } from '~/utils/validations/login'
 import { snackbarVariants } from '~/constants'
 import { styles } from './ResetPassword.styles'
 import imgSuccess from '~/assets/img/email-confirmation-modals/success-icon.svg'
+import AppButton from '~/components/app-button/AppButton'
```
Comment (D1nviz) [inline src/containers/guest-home-page/reset-password/ResetPassword.jsx:23]: move it up to oher components

**Comment by D1nviz** [comment]
Please, sort import according to our convenience

File: src/containers/guest-home-page/reset-password/ResetPassword.jsx (hunk 5 above)
Comment (mhevyk) [reply src/containers/guest-home-page/reset-password/ResetPassword.jsx]: Rearranged imports. While doing it also fixed some typos and small issues, for example error in console caused by `EmptyCooperationActivities` container

**Comment by mhevyk** [comment]
> Please, sort imports according to our convenience

Sorted

**Review by D1nviz — APPROVED** [review]

**Review by Mav-Ivan — APPROVED** [review]

**Review by byavdoshniak — APPROVED** [review]

**Pull request merged**